src/main/java/com/revature/domain/PersonRole.java

```diff
@@ -0,0 +1,79 @@
+package com.revature.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="ie_person_role")
+public class PersonRole {
+	
+	@Id
+	@Column(name="pr_id")
+	private int id;
+	
+	@Column(name = "pr_title")
+	private String title;
+
+	public PersonRole(int id, String title) {
+		super();
+		this.id = id;
+		this.title = title;
+	}
+
+	public PersonRole(){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)